dev-scripts/test-scripts/layout-test.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+pushd crates/layout/tests
+cargo test -- --nocapture --test-threads=1
```

Comment:
Why do we need a script? Can't we run it from the root with the rest of the tests?

Reply:
We don't need a script, but need to run with --test-threads=1. I just did this to make it easier to run this specific test. You are the test expert so let me know what the best way to do this is.

Reply:
That's a bit of a code smell. Why can't we run them in parallel?

Reply:
Each tests adds views to the layout manager and computes layout. If they run in parallel the layout manager has views from all the different tests at the same time and calculates a different layout.

Reply:
Can they each instantiate their own layout manager?

Reply:
I think they should too (also, we could end up with multiple activities on screen at once hosted by the same process, so at that level we need to have some isolation, too); but this might be a bigger interface change because I don't see where the layout manager is instantiated in `setup()` in here...

Reply:
No they can't instantiate a layout manager as it works right now since the layout manager is a lazy_static singleton. I can make a ticket to change this but it will take some time, as we'll need to make JNI function additions and modifications for the client to instantiate one and refer to it for subsequent calls.

Reply:
This is a necessary change IMO, for both reasons. I'd honestly prefer that to be done first, rather than having this one go in with workarounds for the tests not being able to run in parallel.

Reply:
I added a ticket to make it support multiple threads https://github.com/google/automotive-design-compose/issues/475

Reply:
Removed the test script.